quickstart/101-machine-learning/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "name" {
+  type        = string
+  description = "Name of the deployment"
+  default     = "azureml999"
```

Comment:
This should either not have a default or use the [random provider](https://registry.terraform.io/providers/hashicorp/random/latest/docs) to generate a random name.